setup.py

```diff
@@ -21,7 +21,7 @@
 
 sys.path[:] = path_copy
 
-install_requires = ["six>=1.10.0", "promise>=2.1", "rx>=1.6.0"]
+install_requires = ["six>=1.10.0", "promise>=2.1", "rx<3.0.0"]
```

Comment:
Same suggestion as @ProjectCheshire 
`1.6.0 >= rx < 3.0.0`

Reply:
Using environment markers seems like the correct solution. There's no need to limit RX for people actually using Python 3 is there?
```suggestion
install_requires = [
    "six>=1.10.0",
    "promise>=2.1",
    'rx>=1.6.0,<3;python_version<"3"',
    'rx>=1.6.0;python_version>="3"',
]
```

Reply:
Possibly out of scope of this PR, but brings up related dependencies amongst all the repos @dan98765 @SmileyChris

https://github.com/graphql-python/graphql-ws/blob/5f2cff5578b52f6c363567e5252d5e5d48e64787/graphql_ws/observable_aiter.py

Uses the extensionmethod that no longer exists in rx => 3, so the user python won't matter there

Reply:
```suggestion
install_requires = [
    "six>=1.10.0",
    "promise>=2.1",
    'rx>=1.6.0,<3;python_version<"3.6"',
    'rx>=1.6.0;python_version>="3.6"',
]
```

Reply:
I would keep the minimum requirement and make it "rx>=1.6,<3".

Reply:
I just know that this needs to be resolved... rx came out of Beta yesterday and is breaking installations.

Reply:
Ok, I will try to publish a new release as a hotfix.

Reply:
Hotfix release 2.2.1 is out now. Let me know if there are any problems with this release.